README.md

```diff
@@ -20,31 +15,25 @@ src
 ├── pages
 ├── styles
 ├── util
-```
+
 
 ## 👩‍💻 Develop
 
 - 🤝 [Contribution guidelines](.github/CONTRIBUTING.md)
-- 📚
-  [Content and Markdown guidelines](https://gatsby-theme-carbon.now.sh/components/markdown)
-- 🗺
-  [Navigation guidelines](https://gatsby-theme-carbon.now.sh/guides/navigation/sidebar)
+- 📚 [Content and Markdown guidelines](https://gatsby-theme-carbon.now.sh/components/markdown)
+- 🗺 [Navigation guidelines](https://gatsby-theme-carbon.now.sh/guides/navigation/sidebar)
 
-- `yarn install` – install dependencies
+- `yarn install` – installs all project dependencies listed in `package.json`
 - `yarn dev` – start the development server
 - `yarn dev:clean` – use this if you have cache issues
 - `lint:js` – lint your JavaScript files
-- `format` - run Prettier
+- `format` – run Prettier
 
-If you need more detailed information on how to set up your machine to develop
-locally, please take a look at our
-[wiki](https://github.com/carbon-design-system/carbon-website/wiki).
```

Comment:
I'm pretty sure this formatting is straight from prettier and will cause the format check to fail